contrib/download-frozen-image.sh

```diff
@@ -42,6 +42,8 @@ while [ $# -gt 0 ]; do
 	[ "$tag" != "$imageTag" ] || tag='latest'
 	tag="${tag%@*}"
 
+	imageFile="${image////_}" # "/" can't be in filenames :)
```

Comment:
@tianon iirc it had to do with Bash changing to another license at some point, which wasn't compatible with the OS X Licensing? I know it s#cks